src/main/scala/scalaz/analytics/LocalAnalyticsModule.scala

```diff
@@ -135,12 +158,18 @@ trait LocalAnalyticsModule extends AnalyticsModule {
     case class Column(colName: String, rType: Reified)        extends RowFunction
 
     // constants
-    case class IntLiteral(value: Int)            extends RowFunction
-    case class LongLiteral(value: Long)          extends RowFunction
-    case class FloatLiteral(value: Float)        extends RowFunction
-    case class DoubleLiteral(value: Double)      extends RowFunction
-    case class DecimalLiteral(value: BigDecimal) extends RowFunction
-    case class StringLiteral(value: String)      extends RowFunction
+    case class IntLiteral(value: Int)                 extends RowFunction
+    case class LongLiteral(value: Long)               extends RowFunction
+    case class FloatLiteral(value: Float)             extends RowFunction
+    case class DoubleLiteral(value: Double)           extends RowFunction
+    case class DecimalLiteral(value: BigDecimal)      extends RowFunction
+    case class StringLiteral(value: String)           extends RowFunction
+    case class BooleanLiteral(value: Boolean)         extends RowFunction
+    case class ByteLiteral(value: Byte)               extends RowFunction
+    case class NullLiteral(value: Null)               extends RowFunction
```

Comment:
Because there is only 1 of these, I think you don't need to store it, and can just use `case object NullLiteral extends RowFunction`.